### ISSUE TITLE
Duplicate calenders showing in DAVx5 when refreshing calendar list

### ISSUE THREAD

**Issue by kattelk** [go-vikunja/api#51]
### Description

I am not sure if this is a bug in Vikunja or in DAVx5.

When adding my Vikunja account to DAVx5, my Vikunja calendars are shown in the list of calendars.  If I then refresh the list of calendars, each calendar is then duplicated in the list,  showing me double the amount of calendars.

### Vikunja Frontend Version

Does not seem to have a name

### Vikunja API Version

v0.19.2+71-4ec4c0a65d

### Can you reproduce the bug on the Vikunja demo site?

Yes

**Comment by sumitmamoria** [comment]
I am having the same issue.

**Comment by kolaente** [comment]
Not sure why it does this. Vikunja only returns the lists you actually have without doubling them or anything. Tasks.org seems to correctly show lists only once. Still this may be a Vikunja issue if Davx⁵ expects some unique identifier per list which Vikunja does not provide. But given how it works fine with Tasks.org and not with Davx⁵ that seems to suggest it's more the fault of Davx⁵ than Vikunja.
I'd love if someone with more knowledge of Davx⁵ would debug this and point me in the right direction.

**Comment by kolaente** [comment]
Is this still a problem?

**Comment by florianschroen** [comment]
I added a calendar through davx5 and now all calendars are listed twice.

When looking at the calendar properties, the calendars with the same name share the same caldav url. So this seems a display issue, only. 

Using version 4.3.12.1-gplay (403120101), created 2023/12/25

**Comment by kolaente** [comment]
Does the same happen when you re-add the dav connection?

**Comment by florianschroen** [comment]
I guess, yes. Have not tried yet. 

Found my issue in the clear text debug info from davx5. I moved my nextcloud instance to a new url (with redirects) a few months ago.

Maybe this is not relevant to this ticket. Sorry.

**Comment by kolaente** [comment]
Okay, closing this as resolved then. Please ping  again or open a new issue if you can reproduce it reliably.
